### PR TITLE
Completed Integration tests for 'users' and 'home' controller

### PR DESCRIPTION
**Pull request by sunilkr**
#22 

 * Updated doc\developer-guide-docker-compose.md to cover Docker Desktop with WSL2 backend on Windows 10.
 * Renamed users_test -> user_password_test to be consistent with controller. Other user paths are already covered in devise_registration_test, devise_login_test.
 * Added test for root path/home page.

### CODE REVIEW

**Comment by sunilkr** [comment]
@anantshri @abhisek please review.

File: test/integration/users_test.rb [hunk 1]
```diff
@@ -1,44 +0,0 @@
-require 'test_helper'
```
Comment (abhisek) [inline test/integration/users_test.rb:1]: @sunilkr Why are we deleting this file?

**Review by abhisek — APPROVED** [review]
Approved for newly added integration tests

**Review by abhisek — APPROVED** [review]

File: test/integration/users_test.rb (hunk 1 above)
Comment (sunilkr) [reply test/integration/users_test.rb:1]: Renamed users_test -> user_password_test to be consistent with controller. Other user paths are already covered in devise_registration_test, devise_login_test. Git didn't treat as file rename.

**Pull request merged**